src/content/en/2020/accessibility.md

```diff
@@ -226,227 +346,247 @@ Some assistive technologies for the Web include:
 
 These techniques are particularly helpful when something in the visual interface is redundant or unhelpful to assistive technology users. It should be used thoughtfully as it is essential to deliver feature parity for all users. Avoid using it to skip over content that is challenging to make accessible.
 
-Hiding and showing content is a prevalent pattern in modern interfaces, and it can be helpful to declutter the UI for everyone. There are two ARIA attributes that are helpful additions to this disclosure pattern. The `aria-expanded` attribute should have a `true`/`false` value that toggles depending on whether the disclosed content is shown or not. Additionally the `aria-controls` attribute can be associated with an `id` on the disclosed content creating  a programmatic relationship between the triggering control (which should be a button) and the content that gets displayed.
+Hiding and showing content is a prevalent pattern in modern interfaces, and it can be helpful to declutter the UI for everyone. There are two ARIA attributes that are helpful additions to this disclosure pattern. The `aria-expanded` attribute should have a `true`/`false` value that toggles depending on whether the disclosed content is shown or not. Additionally the `aria-controls` attribute can be associated with an `id` on the disclosed content creating a programmatic relationship between the triggering control (which should be a button) and the content that gets displayed.
 
-We found that 20.98% of desktop pages and 21.00% of mobile pages had at least one element with the `aria-expanded` attribute and 17.38% of desktop pages and 16.94% of mobile pages had at least one element with the `aria-controls` attribute. This suggests that around 1/5th of websites might be implementing at least partially accessible disclosure widgets. Note that the `aria-controls` attribute is considered a best practice for the disclosure pattern because screen reader support is not ideal.
+We found that 20.98% of desktop pages and 21.00% of mobile pages had at least one element with the `aria-expanded` attribute and 17.38% of desktop pages and 16.94% of mobile pages had at least one element with the `aria-controls` attribute. This suggests that around one fifth of websites might be implementing at least partially accessible disclosure widgets. Note that the `aria-controls` attribute is considered a best practice for the disclosure pattern because screen reader support is not ideal.
 
-##### Screen reader only text
+{# TODO (Authors) - I don't understand this last sentence. Why is it a best practice because support is not ideal? Are we missing a "not"? Or do you mean it's a best practice "despite" support not being ideal? #}
```

Comment:
@oluoluoxenfree / @alextait1 ?

Reply:
I was added after a comment from @aardrian because we don't want people to think aria-controls is as necessary as aria-expanded. It is best practice to include it, but because SR support isn't 100% it's not as essential to the disclosure pattern

Reply:
Ok but then it’s not “best practice *because* it’s not being widely supported, but *despite* it not being widely supported? I don’t think the implication that a best practice means it’s not required is clear.

Or even “it’s considered best practice, but is not essential, because support...”

Reply:
Updated